gui/gui-manager.cpp

```diff
@@ -547,10 +547,16 @@ void GuiManager::runLoop() {
 
 		redraw();
 
-		// Delay until the allocated frame time is elapsed to match the target frame rate
-		uint32 actualFrameDuration = _system->getMillis(true) - frameStartTime;
-		if (actualFrameDuration < targetFrameDuration) {
-			_system->delayMillis(targetFrameDuration - actualFrameDuration);
+		// Delay until the allocated frame time is elapsed to match the target frame rate.
+		// In case we have vsync enabled, we force a minimum frame duration of 1 millisecond
+		// since otherwise, we'd have a very sluggish cursor on 60Hz displays.
+		if (ConfMan.getBool("vsync")) {
```

Comment:
do not ask for config, but ask backend. since vsync might be actually not enabled

Reply:
Thanks! I hope to have properly adressed this in f628d7b.